docs/networking/istio-lab.md

```diff
@@ -283,54 +519,69 @@ Then, try the request again:
 ```bash
 kubectl exec -it ${CURL_INSIDE_POD} -n pets -- curl -IL store-front.pets.svc.cluster.local:80
 ```
+Expected output:
+
+```bash
+HTTP/1.1 200 OK
+server: envoy
+date: Thu, 10 Jul 2025 17:26:57 GMT
+content-type: text/html
+content-length: 443
+last-modified: Wed, 09 Jul 2025 17:55:47 GMT
+etag: "686ead23-1bb"
+accept-ranges: bytes
+x-envoy-upstream-service-time: 12
+```
 
-This **succeeds**, proving that **only Istio-managed services inside the mesh** can talk to each other.
+This succeeds, proving that **only Istio-managed services inside the mesh** can talk to each other.
 
-So far, the `store-front` service is only accessible **inside the cluster**. To allow **external users** to access it (e.g., from a browser), we need an **Istio Ingress Gateway**.  
+So far, the `store-front` service is only accessible inside the cluster. To allow external users to access it we need an **Istio Ingress Gateway**.
```

Comment:
When deploying the AKS Store Demo application using the aks-store-all-in-one.yaml manifest, it actually deploys store-front and store-admin services using LoadBalancer, so we'll need to update this statement and/or manually set the service type to ClusterIP.

Reply:
makes sense to me. I will add an instruction that after getting the AKS Store Demo we will need to change the service to ClusterIP